packages/api-reference/src/components/Content/Schema/SchemaProperty.vue

```diff
@@ -283,6 +300,14 @@ const rules = ['oneOf', 'anyOf', 'allOf', 'not']
   border-radius: var(--scalar-radius-lg);
 }
 
+.property-enum-values {
+  display: flex;
+  gap: 6px;
+}
+.property-enum-values-long {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
we might use https://developer.mozilla.org/en-US/docs/Web/CSS/repeat

or define is a better approach, for example. Display enum values as chips or code blocks